engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicDeploymentManager.scala

```diff
@@ -210,7 +210,13 @@ class PeriodicDeploymentManager private[periodic] (
       deployedVersionId: Option[VersionId],
       currentlyPresentedVersionId: Option[VersionId],
   ): Future[ProcessState] = {
-    val statusDetails = statusDetailsList.head
+    val statusDetails = statusDetailsList match {
+      case ::(head, _) =>
+        head
+      case Nil =>
+        val status = PeriodicProcessStatus(List.empty, List.empty)
```

Comment:
That is the default value, that was always returned for ScenarioName with no deployments.